bad.rb

```diff
@@ -0,0 +1,7 @@
+# documentation of this class
+class Foo
+  def this_line_is_too_long(thissssss, liiiiiiiiine, iiiiissssss, tooooooo_looong)
```

Comment:
W: Unused method argument - `thissssss`. If it's necessary, use `_` or `_thissssss` as an argument name to indicate that it won't be used. You can also write as `this_line_is_too_long(*)` if you want the method to accept any arguments but don't care about them.
W: Unused method argument - `liiiiiiiiine`. If it's necessary, use `_` or `_liiiiiiiiine` as an argument name to indicate that it won't be used. You can also write as `this_line_is_too_long(*)` if you want the method to accept any arguments but don't care about them.
W: Unused method argument - `iiiiissssss`. If it's necessary, use `_` or `_iiiiissssss` as an argument name to indicate that it won't be used. You can also write as `this_line_is_too_long(*)` if you want the method to accept any arguments but don't care about them.
W: Unused method argument - `tooooooo_looong`. If it's necessary, use `_` or `_tooooooo_looong` as an argument name to indicate that it won't be used. You can also write as `this_line_is_too_long(*)` if you want the method to accept any arguments but don't care about them.
C: Line is too long. [82/80]